app/controllers/bets_controller.rb

```diff
@@ -1,5 +1,22 @@
 class BetsController < ApplicationController
-  def new_bet
+  def new
     @bet = Bet.new
   end
+
+  def create
+    @bet = Bet.new(bet_params)
+    if @bet.save
+      flash[:success] = 'Bet has been created'
+      redirect_to root_path
+    else
+      flash[:danger] = 'Could not create bet'
+      redirect_to create_bet_path
```

Comment:
Should render the create page so that the form fields stay populated with the values the user tried to use.